packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.module.scss

```diff
@@ -0,0 +1,42 @@
+@use '@/scss/underscore' as _;
+
+.placeholder {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: _.unit(5);
+
+  .title {
+    font: var(--font-title-1);
+  }
+
+  .description {
+    font: var(--font-body-2);
+    color: var(--color-text-secondary);
+    margin-top: _.unit(1);
+    text-align: center;
+    max-width: 600px;
+  }
+
+  .options {
+    margin-top: _.unit(6);
+    display: flex;
+    justify-content: space-between;
+    align-items: stretch;
+    max-width: 736px;
+    gap: _.unit(4);
+
+    .option {
+      flex: 1;
+      display: flex;
+      flex-direction: column;
```

Comment:
Looks like the card layout does not need to a flex-box?

Reply:
Need to set the type description content to fullfill the remaining space to make buttons alignment.
<img width="925" alt="image" src="https://user-images.githubusercontent.com/10806653/222338516-bcc040f6-ce94-4b54-8607-da9a3a5ebd2d.png">